pgjdbc/src/test/java/org/postgresql/test/jdbc2/EnumTest.java

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright (c) 2020, PostgreSQL Global Development Group
+ * See the LICENSE file in the project root for more information.
+ */
+
+package org.postgresql.test.jdbc2;
+
+import org.postgresql.test.TestUtil;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+
+import java.sql.Array;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collection;
+
+@RunWith(Parameterized.class)
+public class EnumTest extends BaseTest4 {
```

Comment:
Does it make sense to add the `enumArray` and `enumArrayArray` tests to the  existing `org.postgresql.test.jdbc2.ArrayTest` class?

Reply:
There's overlap between the existing tests, and `Parameterized` tests in JUnit4 require all the tests to have the same parameters.
So I'm inclined to create new test classes (at least for junit4)

Reply:
I am fine either way, but the parameters are the same (BinaryMode).